module_intros/locality.Filter.ipynb

```diff
@@ -0,0 +1,400 @@
+{
```

Comment:
<p>We need to mention what a full/complete neighbor shell is. Is it because, e.g., in SANN, a particle needs to have at least 3 neighbors so that the solid angles add up to 4pi?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/freud-examples/pull/65/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>